app/controllers/podcast_episodes_controller.rb

```diff
@@ -5,14 +5,19 @@ class PodcastEpisodesController < ApplicationController
   def index
     @podcast_index = true
     @podcasts = Podcast.reachable.order("title asc")
-    @podcast_episodes = PodcastEpisode.reachable.order("published_at desc").first(20)
+
+    @podcast_episodes = PodcastEpisode.
+      reachable.
+      includes(:podcast).order("published_at desc").first(20)
     if params[:q].blank?
-      set_surrogate_key_header("podcast_episodes_all " + params[:q].to_s,
-                               @podcast_episodes.map { |e| e["record_key"] })
```

Comment:
`e["record_key"]` is always nil, the actual record key is an attribute of the model object, hence this will generate "podcast_episodes_all" followed by a single empty space. Don't think it's what we want.

```ruby
[1] pry(main)> PodcastEpisode.last["record_key"]
=> nil
[2] pry(main)> PodcastEpisode.last.record_key
=> "podcast_episodes/12"
```

Spaces between keys are already added by fastly-rails:

https://github.com/fastly/fastly-rails/blob/6ed13a694b8753d43515079ae667fe6f8c0fcd23/lib/fastly-rails/action_controller/surrogate_key_headers.rb#L6-L9